service/src/test/java/org/mskcc/cmo/metadb/service/MockDataUtils.java

```diff
@@ -88,7 +88,7 @@ public MetaDbRequest extractRequestFromJsonData(String requestJson) throws Excep
     }
 
     /**
-     * Extracts a List of MetaDbSample's given a request json string;
+     * Extracts a List<MetaDbSample> given a request json string
```

Comment:
this is where the checkstyle issue is failing, i would just revert it to what it was previously